readme.md

```diff
@@ -192,7 +185,25 @@ A couple of examples:
 
 For additional guidance and important information about certificates, please refer to [this blog post](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-changes-are-coming-and-why-you-should-care/ba-p/1658456) from the security team.
 
-## Long-Term Support (LTS)
+## Support
+
+The Azure IoT Hub Device Client supported releases is outlined in the following table.
+
+Refer to the [Azure IoT Device SDK lifecycle and support](https://learn.microsoft.com/azure/iot/iot-device-sdks-lifecycle-and-support) for details on the different supported stages.
+
+| Release | Category | End-of-life |
+|-|-|-|
+| 1.41.3 | Active | - |
```

Comment:
A new version just shippped.

```suggestion
| 1.42.0 | Active | - |
```

Reply:
Oh, is the idea that we continue to add all new versions so that ones previously in the json continue to stay there, at least until the EOL date?

Reply:
Old versions would be deleted when they are out of support.  This would ideally just contain currently supported versions, but with eol it means that it doesnt matter if they are there because it explicitly says its no longer supported.

The eol would be used for:
1. We dont need to update the file when a version goes out of support
2. Azure recommendations can start advising people to update before while the version is in support, so say with 1.42, they could start advising 3 months before support ends.